src/state/stake/hooks.ts

```diff
@@ -893,6 +904,7 @@ export interface DoubleSideStakingInfo extends StakingInfoBase {
   multiplier: JSBI
   // total staked AVAX in the pool
   totalStakedInWavax: TokenAmount
+  totalStackedInUsd: TokenAmount
```

Comment:
Lil typo haha. Should be `totalStakedInUsd`

Reply:
haha, will update it.

Reply:
resolved